packages/app/src/views/Home.vue

```diff
@@ -16,15 +27,121 @@
 <script>
 import CourseSearchBar from "../components/CourseSearchBar";
 import Timetable from "../components/Timetable";
+import TimetableCourseCard from "../components/TimetableCourseCard";
 import COURSES_SEARCH_BAR_QUERY from "../graphql/CoursesSearchBar.gql";
 
 export default {
   components: {
     CourseSearchBar,
-    Timetable
+    Timetable,
+    TimetableCourseCard
   },
   data() {
     return {
+      courseCards: {
+        CSC258H5S: {
+          codeAndName: "CSC258H5 S Computer Organizaion",
```

Comment:
I think we should keep code and name separate. You can use string interpolation to get the behaviour you want like this from Home.vue: 
![image](https://user-images.githubusercontent.com/19757203/77473954-a7042e80-6dec-11ea-811d-31c5d74c1a4f.png)